### PR TITLE
use type Any for hg.If true_child and false_child

### PR DESCRIPTION
**Pull request by wipascal**
often I use strings directly as true_child and false_child

### CODE REVIEW

File: htmlgenerator/base.py [hunk 1]
```diff
@@ -213,8 +213,8 @@ class If(BaseElement):
     def __init__(
         self,
         condition: typing.Union[bool, Lazy],
-        true_child: BaseElement,
-        false_child: typing.Optional[BaseElement] = None,
+        true_child: typing.Any,
+        false_child: typing.Any = None,
```
Comment (saemideluxe) [inline htmlgenerator/base.py:217]: Should ```false_child``` not be ```typing.Optional```?

Comment (wipascal) [reply htmlgenerator/base.py:217]: I think `typing.Optional` is a subtype of `typing.Any`

Comment (wipascal) [reply htmlgenerator/base.py:217]: proof by IDE: 
image

Comment (saemideluxe) [reply htmlgenerator/base.py:217]: Might be. My thought was that Optional[Any] would be more expressiv. It makes clear that the ```If``` can be used without an "else" block. Docs regarding Optional also suggest this type of use (see https://docs.python.org/3/library/typing.html#typing.Optional).

**Review by saemideluxe — APPROVED** [review]

**Pull request merged**